aiutils/tftools/layers.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import tensorflow as tf
+
+
+def identity(input):
```

Comment:
I agree it's probably not that useful unless it were a means of initialization (Identity initialized RNNs for example)? I'd probably take it out in its current form.